morgen: Don't depend on an unsupported electron version

thufschmitt opened the pull request:
###### Description of changes

Remove the explicit dependency of Morgen on electron 15 since it doesn't appear to be needed any more and that electron version is unsupported.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Commit:
morgen: Don't depend on an unsupported electron version

Morgen used to hardcode a dependency on electron 15, which is
unsupported, while it works fine with the latest electron.

ajs124 commented:
Apparently you aren't supposed to just reference `electron`: https://github.com/NixOS/nixpkgs/pull/201768#issuecomment-1320158496

Commit:
morgen: Hardcode the electron version

Apparently the policy for electron packages per https://github.com/NixOS/nixpkgs/pull/201768#issuecomment-1320158496

thufschmitt commented:
> Apparently you aren't supposed to just reference `electron`: [#201768 (comment)](https://github.com/NixOS/nixpkgs/pull/201768#issuecomment-1320158496)

Thanks for the info. I'm not sure that I like that, but let's stick to the convention then. I've just pushed a change hardcoding that to electron 22 (the current latest). As far as I could test morgen works fine with it.

WolfangAukang commented:
Result of `nixpkgs-review pr 206215` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>morgen</li>
  </ul>
</details>

wegank commented:
Superseded by #213972.
